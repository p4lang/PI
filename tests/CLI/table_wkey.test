#!/bin/sh
./run_one_test.py $srcdir/testdata table_wkey simple_router.json
