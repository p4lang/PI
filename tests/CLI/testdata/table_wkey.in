table_add ipv4_lpm 10.0.0.1/12 => set_nhop 10.0.0.1 13
table_dump ipv4_lpm
table_delete_wkey ipv4_lpm 10.0.0.1/12
table_dump ipv4_lpm
table_add ipv4_lpm 10.0.0.1/12 => set_nhop 10.0.0.1 13
table_dump ipv4_lpm
table_modify_wkey ipv4_lpm 10.0.0.1/12 => set_nhop 10.0.0.2 99
table_dump ipv4_lpm
