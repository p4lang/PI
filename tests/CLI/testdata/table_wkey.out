????
Device assigned successfully.
Selecting device.
????
Entry was successfully added with handle 0.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 0
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Action entry: set_nhop - 0a000001, 0013
==========
Dumping default entry
EMPTY
==========
????
Entry was successfully removed.
????
Successfully retrieved 0 entrie(s).
==========
TABLE ENTRIES
==========
Dumping default entry
EMPTY
==========
????
Entry was successfully added with handle 16777216.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 16777216
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Action entry: set_nhop - 0a000001, 0013
==========
Dumping default entry
EMPTY
==========
????
Entry was successfully modified.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 16777216
Match key:
* ipv4.dstAddr        : LPM       0a000001/12
Action entry: set_nhop - 0a000002, 0099
==========
Dumping default entry
EMPTY
==========
????
